### PR TITLE
Update a few things

### PR DESCRIPTION
**Pull request by JJL772**
Basically just pulling in changes from p2ce itself.

also made gameinfo a bit more readable.

### CODE REVIEW

File: cfg/mounts.kv
Comment (AWildErin) [inline cfg/mounts.kv:85]: This needs to be commented out.

File: cfg/mounts.kv
Comment (AWildErin) [inline cfg/mounts.kv:99]: Ditto

File: cfg/mounts.kv
Comment (AWildErin) [inline cfg/mounts.kv:55]: Ditto of below

File: cfg/mounts.kv
Comment (AWildErin) [inline cfg/mounts.kv:165]: nit: Weird empty line?

File: cfg/cvar_defaults.cfg
Comment (AWildErin) [inline cfg/cvar_defaults.cfg:1]: Do we really need to have this here? I don't think this comment needs to transfer over, and probably should retain the old comment above imho.

**Review by AWildErin — CHANGES_REQUESTED** [review]
The main thing that needs to change before this can be merged is the mounts.kv stuff. Other than that it's fine, the other comments are mainly nits

File: cfg/mounts.kv
Comment (JJL772) [reply cfg/mounts.kv:85]: Yeah just realized I copied from my p2ce install

File: cfg/cvar_defaults.cfg
Comment (AWildErin) [inline cfg/cvar_defaults.cfg:33]: This is redundant, it's set later in the file.

**Review by AWildErin — APPROVED** [review]
One final thing! Other than this tiny change it looks good to me, so I'm gonna approve it and once that change is done it can be merged as it's just removal of a duplicated line.

**Pull request merged**
